core/src/main/scala/kafka/tools/JmxTool.scala

```diff
@@ -143,7 +143,7 @@ object JmxTool extends Logging {
 
     if (wait && !foundAllObjects) {
       val missing = (queries.toSet - namesSet).mkString(", ")
-      System.err.println(s"Could not find all requested object names after $waitTimeoutMs ms. Missing $missing")
+      System.err.println(s"Could not find all requested object names after $waitTimeoutMs ms. Missing $missing.")
       System.err.println("Exiting.")
```

Comment:
I added back some of the lines of code that I had unknowingly removed from an earlier commit. Thanks for catching this Vahid.